react/modules/enhancedEcommerceEvents.ts

```diff
@@ -320,7 +312,7 @@ export async function sendEnhancedEcommerceEvents(e: PixelMessage) {
     }
 
     case 'vtex:promotionClick': {
-      const { promotions } = e.data as PromoViewData
```

Comment:
It caught this incorrect casting: `vtex:promotionClick` expects `PromotionClickData`, not `PromoViewData`.